src/services/inplayer.account.service.ts

```diff
@@ -52,6 +52,51 @@ export const getUser = async (): Promise<Customer> => {
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const resetPassword: ResetPassword = async ({ customerEmail, publisherId }) => {
+  try {
+    await InPlayer.Account.requestNewPassword({
+      email: customerEmail,
+      merchantUuid: publisherId || '',
+      brandingId: 0,
+    });
+  } catch {
+    throw new Error('Failed to reset password.');
+  }
+};
+
+export const changePassword: ChangePassword = async ({
+  resetPasswordToken,
+  oldPassword = '',
+  newPassword: password,
+  newPasswordConfirmation: passwordConfirmation = '',
+}) => {
+  try {
+    if (resetPasswordToken) {
+      await changePasswordWithResetToken(resetPasswordToken, password, passwordConfirmation);
+    } else {
+      await changePasswordWithOldPassword(oldPassword, password, passwordConfirmation);
+    }
+  } catch {
+    throw new Error('Failed to change password.');
+  }
+};
+const changePasswordWithResetToken = async (token: string, password: string, passwordConfirmation: string) => {
```

Comment:
I think I'd prefer to expose the 2 non-exported methods here instead of trying to force everything through a single interface. The cleeng change password implementation is the same as `changePasswordWithResetToken`